src/Traits/HasPermissions.php

```diff
@@ -265,11 +265,10 @@ public function getPermissionsViaRoles(): Collection
 
     /**
      * Return all the permissions the model has, both directly and via roles.
-     *
-     * @throws \Exception
```

Comment:
Yes, this was a fragment from #926 which should have been removed during the refactor in #1039

Reply:
@drbyte Same goes for ```hasAnyPermission()``` I believe.